test/integration/about_page_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class AboutPageTest < ActionDispatch::IntegrationTest
+  def assert_about_page_i18n(local)
+    get page_url("about"), locale: local
+
+    assert_response :success
+    assert_select(".t-body a") do |elements|
+      assert_equal 13, elements.count
+    end
```

Comment:
I don't think we need to check this closely, just as long as we get a 200 response and this fails if we miss something (like I did yesterday).